README.md

```diff
@@ -1,21 +1,31 @@
-﻿# Quantum Development Kit Samples #
+﻿# Microsoft Quantum Development Kit Samples #
+ [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/Microsoft/Quantum/master)
 
 These samples demonstrate the use of the Quantum Development Kit for a variety of different quantum computing tasks.
-Each sample is provided as a Visual Studio 2017 C# or F# project under the [`QsharpSamples.sln`](./Samples/QsharpSamples.sln) solution.
-The samples are broken down into four broad categories, each of which is described below.
-Most of the samples consist of a Q# source file with detailed comments explaining the sample and a short classical program (either `Program.cs` or `Program.fs`) to call into Q# operations and functions.
+Most samples are provided as a Visual Studio 2017 C# or F# project under the [`QsharpSamples.sln`](./Samples/QsharpSamples.sln) solution.
+
+Each sample is self-contained in a folder. Most of the samples consist of a Q# source file with detailed comments explaining the sample and a short classical program (either `Program.cs` (C#), `Program.fs` (F#), or `host.py` (Python)) to call into Q# operations and functions.
```

Comment:
I think this edit improves things a lot. One question I'd have, though, is how we should let readers know that samples presented as IQ# Notebooks don't need a host program at all since Jupyter itself acts as a host program in that case.